### PR TITLE
Pybindings

### PR DESCRIPTION
**Pull request by ftomassetti**
This is going to take some time...

The main mess is with Travis and AppVeyor because they do not really support two languages.

Going to play with them using this branch.

### CODE REVIEW

**Comment by ftomassetti** [comment]
Finally convinced AppVeyor to collaborate (build is passing). Squashing here and there and preparing to merge.

@psi29a we basically imported the pyhon bindings previously in pyplatec and inserted in plate-tectonics

After merging we could add a note in project pyplatec specifying it was included in plate-tectonics

**Comment by ftomassetti** [comment]
@psi29a if you have nothing against it I will merge it in the next few days and release the new version of PyPlatec, then we can merge also https://github.com/Mindwerks/worldengine/pull/53

**Comment by psi29a** [comment]
I'll give it a look over tomorrow during lunch. :)

File: .travis.yml [hunk 1]
```diff
@@ -2,31 +2,49 @@ language: cpp
...
+env:
+    - TARGET=cpp
+    - PYVERSION=2.6.9
+    - PYVERSION=2.7.8
+    - PYVERSION=3.2.6
+    - PYVERSION=3.3.6
+    - PYVERSION=3.4.3
 
 before_install:
     - echo "yes" | sudo add-apt-repository ppa:kalakris/cmake
```
Comment (psi29a) [inline .travis.yml]: Why do we need a specific version of cmake? Is this some trickery we should document here?

File: .travis.yml [hunk 2]
```diff
@@ -2,31 +2,49 @@ language: cpp
...
     - echo "yes" | sudo add-apt-repository ppa:ubuntu-toolchain-r/test -y
```
Comment (psi29a) [inline .travis.yml]: Same here, why do we need this ppa?

File: .travis.yml [hunk 3]
```diff
@@ -2,31 +2,49 @@ language: cpp
...
-  - if [ "$CXX" = "g++" ]; then sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-4.8 90; fi
-  - if [ "$CXX" = "g++" ]; then sudo apt-get install libstdc++-4.8-dev; fi
+  - echo "== Install step =="
 
-  # clang 3.4
-  - if [ "$CXX" == "clang++" ]; then sudo apt-get install --allow-unauthenticated -qq clang-3.4; fi
-  - if [ "$CXX" == "clang++" ]; then export CXX="clang++-3.4"; fi   
+script: 
+  - if [ "$TARGET" = "cpp" ]; then $CXX --version ; fi
+  - if [ "$TARGET" = "cpp" ]; then cmake . -G 'Unix Makefiles' ; fi
```
Comment (psi29a) [inline .travis.yml]: Shouldn't we push these out to python and cpp specific scripts? This should get rid of a lot of bash ifs.

**Comment by psi29a** [comment]
Aside from my comments, it looks good. I assume the code is 'OK' and focused just on the framework/support of travis/appveyor/etc.

File: .travis.yml (hunk 1 above)
Comment (ftomassetti) [inline .travis.yml]: I do not remember so I will try to remove it and see if it breaks

Comment (ftomassetti) [inline .travis.yml]: It was superfluous, removed

File: .travis.yml (hunk 2 above)
Comment (ftomassetti) [inline .travis.yml]: removed the cmake specific ppa, commented the other two

**Comment by ftomassetti** [comment]
ok, the Travis file is now cleaned and Travis is passing. AppVeyor should not be touched by the last commits and it was passing so I will go on and merge.
In the meantime GitHub went down :)

**Pull request merged**